patches/v8/revert_api_dcheck-fail_when_we_reenter_v8_while_terminating.patch

```diff
@@ -0,0 +1,142 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: clavin <clavin@DESKTOP-QVNUS47.fastel.com>
+Date: Thu, 7 Dec 2023 16:11:40 -0500
+Subject: Revert "[api] DCHECK-fail when we reenter V8 while terminating"
+
+This reverts commit 3eed17c6a12a0f33639aa5cffb822ebe42931ec8.
```

Comment:
nit: it's imo more helpful here to link to a CL or CRBUG than a commit, as it allows for context to be gained more quickly.

Reply:
Added xrefs to CL and to the `upgrades-followup-issue` in 41416fe4b0b76e6dbf2de0bb113af4d9730b851f